sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/sources/ConsoleWriterSuite.scala

```diff
@@ -34,9 +33,9 @@ class ConsoleWriterSuite extends StreamTest {
     Console.withOut(captured) {
       val query = input.toDF().writeStream.format("console").start()
       try {
-        input.addData(1, 2, 3)
+        input.addData(1, 1, 1)
```

Comment:
why this change?

Reply:
The order of collected messages is not the same as input data any more.
To make the test case working, we should either change input data to same elements, or set `spark.default.parallelism` as 1.

Reply:
It's fixable if we attach the partition id to the commit message of ConsoleSink, but is it worth?  cc @zsxwing @jose-torres

Reply:
Generally I think a streaming sink doesn't need to keep the data order w.r.t. the partition id.

Reply:
Makes sense, but can we set the parallelism to 1 instead? I worry that making all the elements the same is more likely to disguise a bug.